pudb/__init__.py

```diff
@@ -80,7 +80,6 @@ def _get_debugger(**kwargs):
             kwargs.setdefault("stdin", tty_file)
             kwargs.setdefault("stdout", tty_file)
             kwargs.setdefault("term_size", term_size)
-            tty_file.close()
```

Comment:
I agree that the current state is incorrect.

Please either add a comment that we're purposefully leaking the FD, or make a way for `Debugger` to close the FD when done.

Reply:
I know this isn't what you mean but: The file closes when the process ends. We're doing one-time initialization of resources which will be used in the indefinite future, so it seems valid to let reference counting take care of it, to me.

I'll look and see whether I can find a better solution though.